tests/ArrayParserTest.php

```diff
@@ -27,7 +27,7 @@ public function testParser(): void
         $this->assertSame([[0 => 'text'], [0 => null], [0 => '1']], $arrayParse->parse('{{text},{NULL},{1}}'));
         $this->assertSame(
             [[0 => '[",","null",true,"false","f"]'], 1 => ''],
-            $arrayParse->parse('"{"[\",\",\"null\",true,\"false\",\"f\"]"}"')
+            $arrayParse->parse('{{"[\",\",\"null\",true,\"false\",\"f\"]"},""}')
```

Comment:
You change working tests. Why?
If need - add new testcase, without remove old.

Reply:
`'"{"[\",\",\"null\",true,\"false\",\"f\"]"}"'` it is incorrect value for the test.
`'{{"[\",\",\"null\",true,\"false\",\"f\"]"},""}'` it is correct.

Reply:
Even `'{{"[\",\",\"null\",true,\"false\",\"f\"]"},""}'` is incorrect due to the value is multi-dimensional or one-dimensional. But for the test it is ok.

Reply:
How about have splitted test for one- and many-dimensional arrays?

Reply:
Ok, good.